src/main/resources/application.properties

```diff
@@ -1,5 +1,14 @@
 server.port=8080
 
+# settings heroku database
+spring.datasource.maxActive=10
+spring.datasource.maxIdle=5
+spring.datasource.minIdle=2
+spring.datasource.initialSize=5
+spring.datasource.removeAbandoned=true
+spring.jpa.hibernate.ddl-auto=create
```

Comment:
What does ddl-auto do?

Reply:
creates the schema od db, after first deploy i have to change it to update